ChangeLog.txt

```diff
@@ -6,6 +6,8 @@ MAJOR:
 - Redesigned the dejure structure of Yemen and reworked Socotra
 - Reworked many colours across the map, based upon: https://forum.paradoxplaza.com/forum/index.php?threads/wtwsms-project-colouring-the-world.915304/
 - Overhauled the map of Mesopotamia and Western Persia, see on: http://forum.paradoxplaza.com/forum/index.php?threads/wtwsms-presentation-info-news-links.829142/page-4#post-21192710
+- Hypsistarians are no longer their own lonely group
+- Pagan players can now convert to Lucifarianism, Ebiniotoi, Antinomianism, and Egyptian Paganism
```

Comment:
@loup99 

No, I mean converting through a spouse or concubine. Right now, Egyptian Pagans cannot send missionaries to other pagans, so I made a workaround for it.

As for Luciferianism, did @EmperorEnlil single that out for huge flavor, or something? It sounds like it's his own personal project.